### ISSUE TITLE
Feature Request: Redux DevTools integration config

### ISSUE THREAD

**Issue by Matmo10** [qdouble/angular2webpack2-starter#113]
Since this starter includes ngrx, I think this might be a useful feature. 

It turns out ngrx/store is compatible with this Redux dev tools chrome/firefox extension: https://github.com/zalmoxisus/redux-devtools-extension .  And after playing around with it, it has a lot more features and seems generally more useful than the ngrx-store-log-monitor. It's being used in the ngrx-example-app also.

But you have to use `StoreDevtoolsModule.instrumentOnlyWithExtension()` instead of `StoreDevtoolsModule.instrumentStore({...etc})`.

I was thinking in constants.js, we could add something like 'extension' to the dev tools options: 
```
/**
 * specifies which @ngrx dev tools will be available when you build and load
 * your app in dev mode. Options are: monitor | logger | both | extension | none
 */
exports.STORE_DEV_TOOLS = 'extension';
```

Then in store-devtools.module.ts , we'd add another clause - something like this:

```
else if (ENV === 'development' && !AOT && 'extension' === STORE_DEV_TOOLS)
    IMPORTS.push(StoreDevtoolsModule.instrumentOnlyWithExtension());
```

I tried this, but it doesn't seem to work. It does work when I have it directly in my app.module.ts's imports, but I'd prefer to add it conditionally based on ENV, as I described above.

Here's more info: https://github.com/ngrx/store-devtools/issues/27#issuecomment-248495112

Might be nice to have, in my opinion anyways :) Thanks for all the work you've put into this seed so far!

**Comment by qdouble** [comment]
@Matmo10 upload a repo of it being used the way you have it now and I'll see if I can incorporate it

**Comment by Matmo10** [comment]
I played around with it in the ngrx-example app. Here's how they import it:
https://github.com/ngrx/example-app/blob/master/src/app/app.module.ts#L66

It works fine if I import it along side the `StoreDevTools` module in this project, and this is how I'm using it now, but I wanted to turn it off for prod builds:
```
imports: [...etc,
                StoreDevToolsModule,
                StoreDevtoolsModule.instrumentOnlyWithExtension()
               ]
```

I've tried configuring and using/adding to the app's imports it conditionally based on ENV and STORE_DEV_TOOLS but I get inconsistent results. I've tried doing it in both `store-devtools.module.ts` as I mentioned above, and in `app.imports.ts`. Based on Mike Ryan's comment it also seems like it it can't be used simultaneously with the ngrx-store-log-monitor. But half the time it works and half the time it doesn't.


Even when I do this with STORE_DEV_TOOLS set to 'both' or 'monitor': 

```
if (ENV === 'development' && !AOT && !['both', 'monitor'].includes(STORE_DEV_TOOLS)) {
    APP_IMPORTS.push(StoreDevtoolsModule.instrumentOnlyWithExtension());
}
```
it still seems to be enabled sometimes. 


I'll close this for now since I'm not really sure what the correct configuration / implementation looks like, and since I just wanted to mention it as a suggestion, really. I had no idea this extension worked with ngrx until today when I saw it mentioned in the ngrx/store gitter channel.
